WordPress/WordPressStatsWidgets/Remote service/StatsWidgetsService.swift

```diff
@@ -0,0 +1,95 @@
+import WordPressKit
+
+/// Type that wraps the backend request for new stats
+class StatsWidgetsService {
```

Comment:
In order to avoid one of the mistakes we made with WPiOS initially, I wonder if we could create a local Swift package to store the services, instead of including them directly in the Widget target.

I realize that right now it means having to do a bit more work, but I see a few advantages in doing so:

1. It forces decoupling the Widget UI logic from the services and model logic in a very healthy way (and avoid repeating the mistakes we did in WPiOS).
2. If we ever want to separate the logic, it's usually much easier to do so at the beginning, than once the codebase has grown (again, thinking about WPiOS today).
3. It's easier to share the service and model with other targets, if we ever need.
4. Kind of the same as point 3: A future migration to newly designed widgets would be much easier.

Reply:
Great suggestion @diegoreymendez and I agree 100%!!